imap_processing/hit/l1b/hit_l1b.py

```diff
@@ -0,0 +1,165 @@
+"""IMAP-HIT L1B data processing."""
+
+import logging
+from dataclasses import fields
+
+import numpy as np
+import xarray as xr
+
+from imap_processing import utils
+from imap_processing.cdf.global_attrs import ConstantCoordinates
+from imap_processing.cdf.utils import write_cdf
+from imap_processing.hit import hit_cdf_attrs
+from imap_processing.hit.l0.data_classes.housekeeping import Housekeeping
+
+logger = logging.getLogger(__name__)
+
+# TODO review logging levels to use (debug vs. info)
+
+
+def hit_l1b(l1a_dataset: xr.Dataset):
+    """Process HIT data to L1B.
+
+    Parameters
+    ----------
+    l1a_dataset : xarray.Dataset
+        l1A data
+
+    Returns
+    -------
+    xarray.Dataset
+        L1B processed data
+    """
+    # TODO: Check for type of L1A dataset and determine what L1B products to make
+    #   Need more info from instrument teams. Work with housekeeping data for now
+    logical_source = "imap_hit_l1b_hk"
+
+    # Create datasets
+    datasets = []
+    if "_hk" in logical_source:
+        dataset = create_hk_dataset()
+        datasets.append(dataset)
+
+    # Create CDF files
+    logger.info("Creating CDF files for HIT L1B data")
+    cdf_filepaths = []
+    for dataset in datasets:
```

Comment:
I'm not sure how things will work with the batch starter and dependency config file when we have one L1A science file creating three L1B files. I'm not sure if batch start will kick off three separate jobs with a different downstream dependency each time or if they will all be created in the same job. Nothing to change for this, that's something I'm not sure about that we might need to check

Reply:
I think it's ok at the moment since we have not added dependency passed `l1b` level for hit. It just won't kickoff process after this.

Reply:
but can we indent these line in `elif` statement.
```
if hk:
  process housekeping
  append to dataset list
elif science:
   process science data
         append processed data to dataset list
```